api/stories/newStoryRouter.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const { crudOperationsManager } = require('../../lib');
+const Stories = require('./newStoryModel');
+
```

Comment:
code looks great.  I feel that you should comment your code.  Looking at the old 'storyModel', each function has a very descriptive explanation of what it is doing:  
`/**
 * Queries the database for a specific story with given ID
 * @param {number} ID the ID to search for in the database
 * @returns {Promise} a promise that resolves to a story object
 */`
Putting in good comments will ensure that not only you, but others that come after you, will know exactly what you were wanting to accomplish.

Reply:
Thanks! I applied comments for the newStoryModel.

Reply:
I second the comments request.  I think of comments as a common or bridge language in cross-functional and longitudinal teams.  They can help members with a variety of skillsets working concurrently or consecutively get on the same page quickly.